spacy/lang/sl/punctuation.py

```diff
@@ -0,0 +1,64 @@
+from ..char_classes import LIST_ELLIPSES, LIST_ICONS, HYPHENS, LIST_PUNCT, LIST_QUOTES, CURRENCY, UNITS, PUNCT, \
+    LIST_CURRENCY, CONCAT_QUOTES
+from ..char_classes import CONCAT_QUOTES, ALPHA_LOWER, ALPHA_UPPER, ALPHA
+from ..char_classes import merge_chars
+from ..punctuation import TOKENIZER_PREFIXES as BASE_TOKENIZER_PREFIXES
+
+_prefixes = ["•", "\-", "\/", "'"] + BASE_TOKENIZER_PREFIXES
+
+_suffixes = (
+        ["—", "–", "\-", "\:", "\+", "\/", "=", "×", "\$", "•", "¯", "%"]
+        + LIST_PUNCT
+        + LIST_ELLIPSES
+        + LIST_QUOTES
+        + LIST_ICONS
+        + [
+            r"(?<=°[FfCcKk])\.",
+            r"(?<=[0-9])(?:{c})".format(c=CURRENCY),
+            r"(?<=[0-9])(?:{u})".format(u=UNITS),
+            r"(?<=[{al}{e}{p}(?:{q})])\.".format(al=ALPHA_LOWER, e=r"%²\-\+", q=CONCAT_QUOTES, p=PUNCT),
+            r"(?<=[{au}][{au}])\.".format(au=ALPHA_UPPER),
+            r"(?<=[{a}][{au}])\.".format(a=ALPHA, au=ALPHA_UPPER),
```

Comment:
What is the reasoning behind the `(a=ALPHA, au=ALPHA_UPPER)` suffix rule?

It makes the rule preceding it redundant; and the rules we have ended up with mean that:

- a period is split off following any lowercase letter
- a period is split off following an uppercase letter preceded by any other letter

If this is the desired behaviour, it might be worth considering changing `al` to `a` in the fourth rule from the top, as this is simpler than having three separate rules and there are very few situations it would catch that the three rules as they stand would not catch.

Reply:
I have noticed a few words in the dataset ended with a capital for some reason and the default tokenizer doesn't split it. I have just included this rule but I don't know if it reallly makes sense.

e.g. the default tokenizer doesn't split `BiH.` but the dataset splits the dot.

Reply:
I would make the decision based on whether or not you think this a feature of Slovenian that is different from most other languages — then I'd add the rule. If however you think it's just a few specific words like `BiH`, I would add these as tokenizer exceptions.

If we do want this rule, it would be worth considering changing the existing rule to 

`r"(?<=[{a}{e}{p}(?:{q})])\.".format(a=ALPHA, e=r"%²\-\+", q=CONCAT_QUOTES, p=PUNCT),` 
and getting rid of the two rules that follow it as the changed rule would catch everything they catch as well without catching very much else. 

On the other hand, if you extend the suffix patterns as Adriane suggested in the conversation below, it would be fine to just add the new rule and leave the old ones as they are.

Reply:
It is not a feature of Slovenian rather a typo, so I have remove the rule. However I have left the rule that splits the initials because there is quite a few examples in the dataset that do this. I am sorry about the confusing number of commits, I am going back and forth on some of these rules, but I think now it should be OK. However I cannot simply extend the rules because I have modified this one to not include numbers, because this is a feature of Slovenian.

`r"(?<=[{al}{e}{p}(?:{q})])\.".format(al=ALPHA_LOWER, e=r"%²\-\+", q=CONCAT_QUOTES, p=PUNCT)`

The original rule looks like this:

`r"(?<=[0-9{al}{e}{p}(?:{q})])\.".format(al=ALPHA_LOWER, e=r"%²\-\+", q=CONCAT_QUOTES, p=PUNCT)`